advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Commvault Express'
+---
+
+Implementing Commvault Express with EDB Postgres Advanced Server requires the following components:
+
+- EDB Postgres Advanced Server
+- Commvault Express software
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server (EPAS) instance.
+- Commvault Express installed.
+- Obtain the EDB Postgres Advanced Server (EPAS) application path and library directory path.
+- Obtain the credentials that are used to access the EDB Postgres Advanced Server (EPAS).
+- Configure the EDB Postgres Advanced Server (EPAS) archive log directory.
+
+## Configure Commvault Express for EDB Postgres Advanced Server (EPAS)
+
+### Core Setup Wizard
+1. Run the Core Setup Wizard from Commvault Express's Command Centre, the wizard helps to set up a disk storage pool and to modify the server backup plan according to your requirements.
+2. Setup Storage Pool/Disk Storage: From the Welcome page, click Let's get started. On the Disk tab, in the Name box, enter a name for the storage pool.
+3. In the MediaAgent box, accept the default value.
+4. Click Local on the radio button for Type.
+5. In the Backup location box, click Browse to assign a path.
+6. To enable deduplication on the storage, move the Use deduplication toggle key to  the right, and in the Deduplication DB location box, browse to select the path to  the deduplication database.
+7. To move to the next setup option, click Save
+
+<p align="center">
+  <img  alt="Setup Storage Pool/Disk Storage" src="Images/DiskStorageConf.png">
+</p>
+
+8. Creating a Server Backup Plan in Core Setup: In the server backup plan, define where data is stored and how often to back up data.
+9. In the Plan name box,type the name of the plan.
+10. In the Backup destinations section, select the Storge and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup PostgreSQL Database Server
+1. From the navigation pane, go to Protect > Databases.
+2. Click Add server.
+3. Select the database type which in this case is PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd1.png">
+</p>
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
```

Comment:
To connect to what server?  I think this is the Postres database.
We need to be more detailed.